metricbeat/module/aerospike/aerospike_test.go

```diff
@@ -143,6 +164,66 @@ func TestParseClientPolicy(t *testing.T) {
 			expectedClientPolicy: ClusterNamePolicy,
 			expectedErr:          nil,
 		},
+		{
+			Name: "Username and password are honored",
+			Config: Config{
+				User:     sampleUser,
+				Password: samplePassword,
+			},
+			expectedClientPolicy: UserPasswordClientPolicy,
+			expectedErr:          nil,
+		},
+		{
+			Name: "Username is set and password is not set",
+			Config: Config{
+				User: sampleUser,
+			},
+			expectedClientPolicy: as.NewClientPolicy(),
+			expectedErr:          nil,
+		},
+		{
+			Name: "Password is set and user is not set",
+			Config: Config{
+				Password: samplePassword,
+			},
+			expectedClientPolicy: as.NewClientPolicy(),
+			expectedErr:          nil,
```

Comment:
I don't know much about Aerospike, but shouldn't it be an error case? Just setting the password?